docs/mkdocs/documentation/deploy_kmod.md

```diff
@@ -60,6 +60,9 @@ Once loaded, kernel modules have all possible permissions to do any kind of oper
 KMM creates privileged workload to load the kernel modules on nodes.
 That workload needs `ServiceAccounts` able to use the `privileged` `SecurityContextConstraint` (SCC).
 
+Note:
+Some namespaces have the [pod-security-admission-syncronization](https://docs.openshift.com/container-platform/4.12/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-opting_understanding-and-managing-pod-security-admission) disabled by default so although it is not recommended for production, using those namespaces without using any powerful `ServiceAccount` should work out of the box.
+
```

Comment:
```suggestion
```

Reply:
I would instead edit the admonition below:

```markdown
!!! warning Some namespaces are trusted

    When setting up RBAC permissions, keep in mind that any user or ServiceAccount creating a `Module` resource in a
    namespace that is part of the [cluster payload](https://docs.openshift.com/container-platform/4.12/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-opting_understanding-and-managing-pod-security-admission)
    or in `openshift-kmm` will result in KMM automatically running privileged workload on potentially all nodes in the
    cluster.
```

Reply:
Fixed.